src/app.py

```diff
@@ -94,13 +91,14 @@ def gather_magazines():
         len_data = len(data)
         parse_counter = 1
         for i in range(1, len_data):
-            if data[i][7] != "1" and data[i][0] != "":
-                p = list(
-                    filter(lambda p: p["slug"] == data[i][2], publications_serialized)
-                )
-                p = p[0] if p else None
+            parsed = data[i][7]
```

Comment:
nit: I think it'd make more sense if `parsed = data[i][7] == "1"` here and you check `if not parsed and ...`